### PR TITLE
fixes people "walking over the glass shard!" when they're on the ground, changes message.

### PR DESCRIPTION
**Pull request by Militaires**
:cl: BASILMAN YOUR MAIN MAN
fix: fixes people "walking over the glass shard!" when they're on the ground, changes message when incapacitated
/:cl:

only because i dont feel good when i dont make a pr for long.

### CODE REVIEW

**Comment by MMMiracles** [comment]
wouldn't being forcefully dragged over shards hurt more than accidentally stepping on them?

also hugbox

**Comment by ExcessiveUseOfCobblestone** [comment]
H U G B O X

**Comment by KorPhaeron** [comment]
Why?

**Comment by Militaires** [comment]
u wot m8

cause it didnt feel right making a "STEPS ON THE GLASS" when you're lying buttnaked on the floor, i can make it have a different message if you're all MUH HUGBOX about this.

**Comment by PKPenguin321** [comment]
sure go ahead and make a different message if you want

**Comment by WJohn** [comment]
Rip bullying digitigrade lizards to death with a single glass shard.

**Comment by AdamElTablawy** [comment]
make it a different message but no it's fucking glass you should have worn shoes moron

**Comment by MMMiracles** [comment]
probably should remove the stun lock if you're incapacitating but leave the damage with a different message instead of outright removing it.

**Comment by Militaires** [comment]
okay done i fixed it

File: code/game/objects/items/stacks/sheets/glass.dm [hunk 1]
```diff
@@ -360,6 +360,12 @@
...
-					cooldown = world.time
+					if(!H.incapacitated())
+						H.visible_message("<span class='danger'>[H] steps in the broken glass!</span>", \
+								"<span class='userdanger'>You step in the broken glass!</span>")
+						cooldown = world.time
+					else
+						H.visible_message("<span class='danger'>[H] slides on the broken glass!</span>", \
+								"<span class='userdanger'>You slide on the broken glass!</span>")
+
+						cooldown = world.time
```
Comment (optimumtact) [inline code/game/objects/items/stacks/sheets/glass.dm]: you don't need the cooldown twice mate.

just put it at the bottom of the if check like so

	if(!H.incapacitated())
		H.visible_message("<span class='danger'>[H] steps in the broken glass!</span>", \
			"<span class='userdanger'>You step in the broken glass!</span>")

	else
		H.visible_message("<span class='danger'>[H] slides on the broken glass!</span>", \
						"<span class='userdanger'>You slide on the broken glass!</span>")

	cooldown = world.time

Comment (Militaires) [inline code/game/objects/items/stacks/sheets/glass.dm]: @optimumtact oh fuck thanks im an autist

i could swear i was looking at that and sayin "there is something fukken wrong" and just prd it

**Pull request merged**